### ISSUE TITLE
Error in log (4.1.2).

### ISSUE THREAD

**Issue by WarC0zes** [j-a-n/lovelace-wallpanel#123]
Hi,
I just updated to version 4.1.2, it fixes the bonton menu fine but I get an error in the log.

```
2023-06-16 05:48:29.147 ERROR (MainThread) [frontend.js.latest.202306080] http://192.168.1.xxx:8123/frontend_latest/51643-TYdZS6HDf8Q.js:1:30165 Uncaught TypeError: Cannot read properties of undefined (reading 'editMode')
```
![wallpaper error](https://github.com/j-a-n/lovelace-wallpanel/assets/91471397/b70baac1-9ac8-4bb8-9fc3-eb3d22fa9a89)
![wallpaper error2](https://github.com/j-a-n/lovelace-wallpanel/assets/91471397/138e979a-cf38-4159-9ee1-de01ae3b507d)

if I go back to version 4.11.1, I no longer get the error.

my code:
```
wallpanel:
  enabled: enable
  hide_toolbar: false
  hide_sidebar: false
  fullscreen: false
  screensaver_stop_navigation_path: /lovelace/default_view
  image_order: sorted
  image_list_update_interval: 3600
  image_fit: cover
  image_url: >-
    https://api.unsplash.com/photos/random?client_id=_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx__fxXY&query=nature
  idle_time: 60
  show_progress_bar: true
  info_move_pattern: random
  info_move_interval: 30
  cards:
```

Is it a configuration issue or a version 4.1.2 issue?

**Comment by j-a-n** [comment]
Thanks for reporting!
Should be fixed with in version 4.12.1.

**Comment by WarC0zes** [comment]
Thanks for this quick fix. No more errors with 4.1.2.1.
